app/post.go

```diff
@@ -158,7 +158,11 @@ func (a *App) CreatePost(post *model.Post, channel *model.Channel, triggerWebhoo
 
 	var pchan store.StoreChannel
 	if len(post.RootId) > 0 {
-		pchan = a.Srv.Store.Post().Get(post.RootId)
```

Comment:
Probably you need to instantiate the channel here (with `make`)

Reply:
yeah! That should probably fix the tests.